client/gutenberg/extensions/utils/get-active-style-name.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import TokenList from '@wordpress/token-list'; // @TODO: Make sure dep is declared Jetpack-side
```

Comment:
> Make sure dep is declared Jetpack-side

It is :thumbsup:

https://github.com/Automattic/jetpack/blob/5ad79289c44abf33726c3853d7683bad0b13ae40/modules/tiled-gallery/tiled-gallery.php#L36-L42